.github/workflows/release-news.yml

```diff
@@ -0,0 +1,36 @@
+name: Release News Workflow
+
+on:
+  # Allow manually starting the workflow incase we need human intervention.
+  workflow_dispatch:
+
+  # Runs at 08:17 UTC / 00:17 PST / 01:17 PDT
+  # 17 is an arbitrarily chosen to avoid running the workflow during the peak at the beginning of the hour
+  schedule:
+    - cron: '17 8 * * *'
```

Comment:
Is arbitrary from your meat brain, dice, or what? :D `echo $(($RANDOM % 60))` if in doubt :D

This immediately inspired me to look up whether it was possible to jitter a GitHub Actions schedule- I didn't find anything.

Reply:
From my brain.

I tried making a variable schedule, but the `cron` value needs to be a static string or else GitHub says it's an invalid workflow.